react/spring-page/client/src/components/Main/Main.js

```diff
@@ -25,44 +21,48 @@ export function Main() {
     const fetchData = async () => {
       const result = await API.get('/info/spring');
       setData(result.data);
+      console.log(isLoggedIn);
+      console.log('first time');
     };
     fetchData();
   }, []);
 
-  return (
-    <main className="main">
-      <Description />
-      <div className="content">
-        <label className="custom-input" htmlFor="search">
-          <input
-            type="text"
-            id="search"
-            placeholder="Input your words"
-            value={name}
-            onChange={(e) => setName(e.target.value)}
-          />
-        </label>
-        <div className="container">
-          <div className="content__info">
-            {data && data.length !== 0 ? (
-              data.map((item, idx) => (
-                // eslint-disable-next-line react/no-array-index-key
-                <a href="/#" key={`сard - ${idx}`} className="content__info__item">
-                  <div className="content__info__item-picture">
-                    <img alt="logo-of-item" src={item.image} />
-                  </div>
-                  <div className="content__info__item-description">
-                    <h3>{item.name}</h3>
-                    <p>{item.description}</p>
-                  </div>
-                </a>
-              ))
-            ) : (
-              <p>There is no result</p>
-            )}
+  if (isLoggedIn) {
```

Comment:
Do this before render and do not deep nest. You can omit else clause if `if` clause has `return`.